### PR TITLE
Removed the unnecessary softplus in NTMHeadBase._address_memory

### PR DESCRIPTION
**Pull request by JulesGM**
Removed the softplus in the softmax:
```python
        s = F.softmax(F.softplus(s), dim=1)
```
softmax already constrains the values to (0, 1), the softplus doesn't achieve anything. Pytorch's softmax implementation is already numerically stable, so that's not the preoccupation.

### CODE REVIEW

**Comment by loudinthecloud** [comment]
Makes sense, thanks for that. Can you please run the copy-task notebook and see we're getting the same results?

**Comment by JulesGM** [comment]
I mean, I trained a bunch of models with this code base and got fine looking gifs, if that's what you're asking

**Comment by JulesGM** [comment]
I trained a bunch of pretty long models, and get good results in the notebooks.

**Comment by JulesGM** [comment]
Like this one, which was trained for a while on sequences up to 120 long, and converges very sharply
 
copy-train-120

**Comment by loudinthecloud** [comment]
Tested it as well, seems to alter convergence a bit but perhaps for the better.
